ruby-challenges.rb

```diff
@@ -11,10 +23,23 @@
 # Expected output: '221 is odd'
 
 # Pseudo code:
+# Created a method called even_or_odd.
+# Inserted the argument reposts1.
+# Inside the method I used an if statement to determine if the number is even or odd.
+# If the number is even it prints a string showing that the number is even. Otherwise, it prints a string showing that the number is odd.
+
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+def remove_vowels(beatles_album1)
```

Comment:
Logic looks great!  Consider making your parameter name something more versatile and relative to the data type being passed in.